sdk/java/src/main/java/com/chain/http/Client.java

```diff
@@ -633,13 +658,96 @@ public Builder setAccessToken(String accessToken) {
     }
 
     /**
+     * Sets the client's certificate and key for TLS client authentication.
+     * @param certPath filepath to pem encoded X.509 certificate
+     * @param keyPath filepath to pem encoded X.509 private key
+     */
+    public Builder setX509KeyPair(String certPath, String keyPath) throws HTTPException {
```

Comment:
How do you feel about taking a bytestream for both of these, rather than doing the file-opening in the client. Or, you can make an overload that does the file opening for the user if they want.